ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3597,6 +3597,33 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "tracing-create"
```

Comment:
Has this name been left as the old name on purpose? (same with destroy)

Reply:
Name changed to observer